docs/dev/new_check_howto.md

```diff
@@ -21,11 +21,11 @@ These requirements are used during the code review process as a checklist. This
 
 Python 2.7 needs to be available on your system. It is strongly recommended to create and activate a [Python virtual environment][5] in order to isolate the development environment. See the [Python Environment documentation][6] for more information.
 
-You'll also need `docker-compose` in order to run the test harness. 
+You'll also need [cookiecutter][1] to generate a template, and `docker-compose` in order to run the test harness. 
```

Comment:
cookiecutter is installed when they run `pip install -r requirements-dev.txt` at line 37